checklist/CheckListCell.swift

```diff
@@ -1,11 +1,12 @@
 import UIKit
 
 
-let checkListCell = "CheckListCell"
-
 class CheckListCell: UITableViewCell {
-  
-  let label:UILabel = {
```

Comment:
using nib/storyboard and `awakeFromNib()` and `prepareForReuse()` will help you manage a lot of the lifecycle/layout issues with subviews.